### ISSUE TITLE
Android: NullPointerException in Preferences

### ISSUE THREAD

**Issue by rtsisyk** [omapsapp/omapsapp#66]
```
Caused by java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.SharedPreferences android.content.Context.getSharedPreferences(java.lang.String, int)' on a null object reference
       at com.mapswithme.util.SecureStorage.load(SecureStorage.java:30)
       at com.mapswithme.maps.LightFramework.nativeIsAuthenticated(LightFramework.java)
       at com.mapswithme.maps.background.NotificationService.notifyIsNotAuthenticated(NotificationService.java:46)
       at com.mapswithme.maps.background.NotificationService.lambda$2_dkPJ00Hno6JDdUGA5HhTCFPWA()
       at com.mapswithme.maps.background.-$$Lambda$NotificationService$2_dkPJ00Hno6JDdUGA5HhTCFPWA.tryToNotify(:2)
       at com.mapswithme.maps.background.NotificationService.tryToShowNotification(NotificationService.java:130)
       at com.mapswithme.maps.background.NotificationService.onHandleWork(NotificationService.java:102)
       at androidx.core.app.JobIntentService$CommandProcessor.doInBackground(JobIntentService.java:392)
       at androidx.core.app.JobIntentService$CommandProcessor.doInBackground(JobIntentService.java:383)
       at android.os.AsyncTask$3.call(AsyncTask.java:378)
       at java.util.concurrent.FutureTask.run(FutureTask.java:266)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
       at java.lang.Thread.run(Thread.java:919)
```

```

I/MapsmeCore: platform/Platform.cpp:258 SetSettingsDir()  Settings path =  /storage/emulated/0/MapsWithMe/
```

https://console.firebase.google.com/u/0/project/omapsapp/crashlytics/app/android:app.omaps.beta/issues/cab267e5eba7bf892b8e32e07431c07e?time=last-seven-days&sessionEventKey=602BD37B000B000176B7DDA5F044E4EF_1508311180746332457

**Comment by rtsisyk** [comment]
Replaced by https://github.com/omapsapp/omapsapp/issues/86

**Comment by rtsisyk** [comment]
```
java.lang.RuntimeException: 
  at android.os.AsyncTask$4.done (AsyncTask.java:415)
  at java.util.concurrent.FutureTask.finishCompletion (FutureTask.java:383)
  at java.util.concurrent.FutureTask.setException (FutureTask.java:252)
  at java.util.concurrent.FutureTask.run (FutureTask.java:271)
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1167)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:641)
  at java.lang.Thread.run (Thread.java:923)
Caused by: java.lang.NullPointerException: 
  at com.mapswithme.util.SecureStorage.load (SecureStorage.java:30)
  at com.mapswithme.maps.LightFramework.nativeIsAuthenticated (LightFramework.java)
  at com.mapswithme.maps.background.NotificationService.notifyIsNotAuthenticated (NotificationService.java:46)
  at com.mapswithme.maps.background.NotificationService.lambda$2_dkPJ00Hno6JDdUGA5HhTCFPWA (NotificationService.java)
  at com.mapswithme.maps.background.-$$Lambda$NotificationService$2_dkPJ00Hno6JDdUGA5HhTCFPWA.tryToNotify (Unknown Source:2)
  at com.mapswithme.maps.background.NotificationService.tryToShowNotification (NotificationService.java:128)
  at com.mapswithme.maps.background.NotificationService.onHandleWork (NotificationService.java:100)
  at androidx.core.app.JobIntentService$CommandProcessor.doInBackground (JobIntentService.java:392)
  at androidx.core.app.JobIntentService$CommandProcessor.doInBackground (JobIntentService.java:383)
  at android.os.AsyncTask$3.call (AsyncTask.java:394)
  at java.util.concurrent.FutureTask.run (FutureTask.java:266)
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1167)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:641)
  at java.lang.Thread.run (Thread.java:923)
  ```
  
  https://play.google.com/console/u/1/developers/8084273541548442467/app/4974916572853654059/vitals/crashes/d4e38e72/details?installedFrom=PLAY_STORE&days=30

**Comment by rtsisyk** [comment]
Fixed on 10.6.0.
